### PR TITLE
Completed Courses component on User page

### PR DESCRIPTION
**Pull request by DrShpongle**
Image 2022-09-14 at 14 24 18

Image 2022-09-14 at 14 38 49

evan peters lesson learned GIF by Team Coco

### CODE REVIEW

File: src/pages/user/components/completed-courses.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+              collection: Collection
+              completed_at: string
+            }) => {
+              return (
+                <div
+                  key={collection.slug}
+                  className="flex border-b border-gray-200 dark:border-gray-800 py-3 items-center space-x-2"
+                >
+                  <Link href={collection.path}>
+                    <a className="blok shrink-0 w-8 h-8 relative">
```
Comment (zacjones93) [inline src/pages/user/components/completed-courses.tsx:99]: is this supposed to be `block`?

File: src/pages/user/components/completed-courses.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                        src={collection.image}
+                        alt=""
+                        objectFit="contain"
+                        layout="fill"
+                      />
+                    </a>
+                  </Link>
+                  <div className="grow">
+                    <Link href={collection.path}>
+                      <a className="blok shrink-0 w-8 h-8 relative dark:hover:text-blue-300 hover:text-blue-700 duration-100">
```
Comment (zacjones93) [inline src/pages/user/components/completed-courses.tsx:110]: and same here, `block`?

File: src/pages/user/components/completed-courses.tsx [hunk 3]
```diff
@@ -64,7 +78,7 @@ const CompletedCourses: React.FC<any> = ({completeCourseData}) => {
                   <Link href={collection.path}>
                     <a className="blok shrink-0 w-8 h-8 relative">
                       <Image
-                        src={collection.image}
+                        src={collection.image as string}
```
Comment (DrShpongle) [inline src/pages/user/components/completed-courses.tsx]: 👍

**Review by zacjones93 — APPROVED** [review]

File: src/pages/user/components/completed-courses.tsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  lesson_count: number
+}
+
+const byDate = (a: CourseData, b: CourseData) =>
+  a.completed_at > b.completed_at ? -1 : 1
+
+const byTitle = (a: CourseData, b: CourseData) =>
+  a.collection.title.localeCompare(b.collection.title)
+
+const CompletedCourses: React.FC<{completeCourseData: CourseData[]}> = ({
```
Comment (vojtaholik) [inline src/pages/user/components/completed-courses.tsx:27]: @DrShpongle components don't belong to `src/pages` directory.

**Pull request merged**